.github/ISSUE_TEMPLATE/new-dbt-test.md

```diff
@@ -0,0 +1,49 @@
+# New DBT Test
+
+## Issue Summary
+
+_Provide a brief description of the test and its motivation._
+
+## Prerequisites
+
+Before proceeding, ensure you have a basic understanding of DBT tests. The [official DBT documentation on data tests](https://docs.getdbt.com/docs/build/data-tests) is a great resource.
+
+## Checklist
+
+Our workflow focuses on creating reusable generic test templates in the `dbt/tests/generic/` directory, which are invoked in the `schema.yaml` file for a given data model. Follow these steps to add a new test:
+
+1. **Check for Existing Tests:** Ensure the test or a similar functionality does not already exist among our generic templates. Consider modifying an existing template if it closely aligns with the required functionality.
+
+2. **Create Test Template:**
+   - [ ] Add a new test template in the `dbt/tests/generic/` directory.
+   - Example: View an existing DBT test template [here](https://github.com/ccao-data/data-architecture/blob/master/dbt/tests/generic/test_unique_combination_of_columns.sql).
+   - _We use [Jinja](https://jinja.palletsprojects.com/en/3.1.x/templates/) to parameterize the generic test templates._
```

Comment:
**[Nitpick]** I think we need some kind of conditional branching here to indicate the different steps the reader should take based on which of the following situations is true:

* A generic test exists that meets the reader's needs
* A generic test exists that _almost_ meets the reader's needs, but must be extended to support their use case
* No generic test exists that meets the reader's needs

Currently these docs imply that the third bullet is the most important step, but I actually think the third bullet should only be a last resort, because writing a new generic requires advanced understanding of dbt.

Reply:
Ah right, because this issue could still be invoked to add an existing generic test to the `schema.yaml` for a given data model.